angular-chess/src/app/types/pieces.t.ts

```diff
@@ -1,21 +1,12 @@
 import { Board } from "./board.t";
```

Comment:
Please normalize the file names with type

Reply:
Is the file name pieces here correct? Please extract the move and FullMove to extra file.